common/src/main/java/org/opensearch/ml/common/MLConfig.java

```diff
@@ -112,9 +119,15 @@ public static MLConfig parse(XContentParser parser) throws IOException {
                 case TYPE_FIELD:
                     type = parser.text();
                     break;
+                case CONFIG_TYPE_FIELD:
+                    configType = parser.text();
+                    break;
                 case CONFIGURATION_FIELD:
                     configuration = Configuration.parse(parser);
                     break;
+                case ML_CONFIGURATION_FIELD:
```

Comment:
Another field added in 2.13, `LAST_UPDATE_TIME_FIELD`, let's use similar way to fix?